tools/check-openapi-helper

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+import os
+import sys
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, ZULIP_PATH)
+os.chdir(ZULIP_PATH)
+from zerver.openapi.openapi import OpenAPISpec, OPENAPI_SPEC_PATH
+EXCLUDE = ["/real-time"]
+VALID_TAGS = ["users", "server_and_organizations", "authentication",
+              "real_time_events", "streams", "messages", "users",
+              "webhooks"]
+def check_file() -> None:
+    openapi_spec = OpenAPISpec(OPENAPI_SPEC_PATH)
+    openapi_spec = openapi_spec.spec()
+    openapi_spec = openapi_spec["paths"]
+    for path in openapi_spec:
+        if path in EXCLUDE:
+            continue
+        for method in openapi_spec[path]:
+            # Check if every file has an operationId
+            if "operationId" not in openapi_spec[path][method]:
+                print("Error: One or more endpoints don't have operationId")
+                return
+            # Check if every file has a valid tag
+            if "tags" not in openapi_spec[path][method]:
+                print("Error: One or more endpoints don't have tags.")
+                return
+            tag = openapi_spec[path][method]["tags"][0]
+            if tag not in VALID_TAGS:
+                print("Error: One of the tags is not a valid tag.")
+check_file()
```

Comment:
Hmm, I think given that this is Python code, it'd be better for this to just be a test in `zerver/tests/test_openapi.py`, which does other validation of the OpenAPI file in Python.  Doing so would save extra parsings of the OpenAPI file.

Reply:
Changed accordingly.